README.md

```diff
@@ -99,4 +99,49 @@ $ docker-compose run bintest
 * [mruby-eso-research](https://github.com/hone/mruby-eso-research) - an app for managing crafting research in Elder Scrolls Online. It uses YAML as the data store.
 
 ## mruby-cli Development
-This app is built as a mruby-cli app, so you just need to run: `docker-compose run compile` and find the binaries in the appropriate directories.
+
+### Compile the mruby-cli binaries
+
+This app is built as a `mruby-cli` app, so you just need to run:
+
+```
+docker-compose run compile
+```
+
+and find the binaries in the appropriate directories.
+
+#### Common error
+
+If rather than typing `docker-compose run compile`, you type `rake compile`, you could get on a nix OS the following error:
```

Comment:
I eventually got that the cross-toolchain is in the container, but I don't at first.

For the text, what do you think of:

```
This app is built as a `mruby-cli` app. To compile the binaries, you **must** type

docker-compose run compile

and find the binaries in the appropriate directories (mruby/build/<target>/bin/).

The docker container contains the necessary cross toolchain to compile a binary for each supported target. Just using `rake compile` will not work out of the box because the main build is designed to compile on a 64-bit Linux host. It could work if you are on a 64-Linux host and you have an cross toolchain equivalent to the one we provide into the docker container.
```

> Just using rake compile will not work because the main Build is designed to compile on a 64-bit Linux Host.

but it will work if I have a equivalent cross-toolchain available in my path and I'm on a 64-bit. right?

If so, maybe rather than just making it clearer in the documentation, I would warn the user. In that way, I'd consider two possible options:

* when using one or another rake task requiring the container, checking that we are into the right container
* when using `compile` task (or any other task requiring it), checking that a compatible toolchain is available